### ISSUE TITLE
Error downloading object for document images

### ISSUE THREAD

**Issue by umdreamer** [lgsvl/Documentation#1]
When using git clone this repository,  there are the errors as follows:

Error downloading object: docs/images/2d-ground-truth-visualized.png (968f12a): Smudge error: Error downloading docs/images/2d-ground-truth-visualized.png (968f12aecaa300b7b2fe64e114360cc74c8ede006ceb760789a9335fc1e94c32): [968f12aecaa300b7b2fe64e114360cc74c8ede006ceb760789a9335fc1e94c32] Object does not exist on the server: [404] Object does not exist on the server

I checked the files at **docs/images/2d-ground-truth-visualized.png**, and it shows:
```
version https://git-lfs.github.com/spec/v1
oid sha256:968f12aecaa300b7b2fe64e114360cc74c8ede006ceb760789a9335fc1e94c32
size 184926
```
It seems the authors did not upload the images onto the github. 

Could anyone can help? Thank you.

**Comment by luqiang21** [comment]
@umdreamer Did you install git lfs?

**Comment by umdreamer** [comment]
> @umdreamer Did you install git lfs?

Yes, I installed git. But the file is just a text file with uiid, not a real png. Did you upload the real png onto the repository?

**Comment by luqiang21** [comment]
@umdreamer git lfs is not as git. You need to install it separately.

**Comment by umdreamer** [comment]
I haved used `git lfs pull`, but it still had errors. 
I used a workaround method by downloading these files seperately from other places. This worked on apollo and autoware data. But this project I cannot find the file.

**Comment by luqiang21** [comment]
@umdreamer I tried to fresh clone with git lfs installed and got similar error. Thank you for reporting. We will fix this.

**Comment by umdreamer** [comment]
@luqiang21 Thank you.

**Comment by bshin-lge** [comment]
This issue has now been fixed. Thanks for the catch!
